website/src/components/Navigation/OrganismNavigation.tsx

```diff
@@ -0,0 +1,89 @@
+
+import { NavigationTab } from './NavigationTab';
+import type { Organism } from '../../config';
+
+interface OrganismNavigationProps {
+    currentOrganism?: Organism;
+    knownOrganisms: Organism[];
+}
+
+export const OrganismNavigation: React.FC<OrganismNavigationProps> = ({ currentOrganism, knownOrganisms }) => {
+    const displayName = 'Organisms';
+
+    return (
+        <Menu as='div' className='relative'>
+            <MenuButton as={NavigationTab} isActive={!!currentOrganism} className='group'>
+                <span>{displayName}</span>
+                {currentOrganism && (
+                    <span className='hidden lg:inline-flex items-center px-2 py-0.5 text-xs font-medium rounded-full bg-primary-50 text-primary-700 border border-primary-200'>
+                        {currentOrganism.displayName}
+                    </span>
+                )}
+                <svg
+                    className='w-4 h-4 transition-transform group-data-[open]:rotate-180'
+                    fill='none'
+                    stroke='currentColor'
+                    viewBox='0 0 24 24'
+                    aria-hidden='true'
+                >
+                    <path strokeLinecap='round' strokeLinejoin='round' strokeWidth={2} d='M19 9l-7 7-7-7' />
+                </svg>
+            </MenuButton>
+
+            <Transition
+                as={Fragment}
+                enter='transition ease-out duration-100'
+                enterFrom='transform opacity-0 scale-95'
+                enterTo='transform opacity-100 scale-100'
+                leave='transition ease-in duration-75'
+                leaveFrom='transform opacity-100 scale-100'
+                leaveTo='transform opacity-0 scale-95'
+            >
+                <MenuItems className='absolute left-0 mt-1 w-56 bg-white rounded-lg shadow-lg border border-gray-200 z-50 focus:outline-none'>
+                    <div className='py-1'>
+                        {knownOrganisms.map((organism) => {
+                            const isActive = currentOrganism?.key === organism.key;
+
+                            return (
+                                <MenuItem key={organism.key}>
+                                    {({ focus }) => (
+                                        <a
+                                            href={`/${organism.key}/search`}
+                                            className={`
+                                                flex items-center gap-3 px-4 py-2 text-sm transition-colors
+                                                ${
+                                                    isActive
+                                                        ? 'bg-primary-100 text-gray-900 font-semibold'
+                                                        : 'text-gray-700'
+                                                }
+                                                ${focus && !isActive ? 'bg-gray-50 text-gray-900' : ''}
```

Comment:
At other locations in the PR, the classes were first stored in separate variables such as `baseClasses` and `stateClasses`. I found that quite readable, would it make sense to do it here as well?